assets/shared/styles/wp-components.scss

```diff
@@ -6,64 +6,13 @@
-@import '~@wordpress/components/src/checkbox-control/style.scss';
-@import '~@wordpress/components/src/circular-option-picker/style.scss';
-@import '~@wordpress/components/src/color-indicator/style.scss';
-@import '~@wordpress/components/src/color-picker/style.scss';
-@import '~@wordpress/components/src/custom-gradient-picker/style.scss';
-@import '~@wordpress/components/src/custom-select-control/style.scss';
-@import '~@wordpress/components/src/dashicon/style.scss';
-@import '~@wordpress/components/src/dimension-control/style.scss';
-@import '~@wordpress/components/src/disabled/style.scss';
-@import '~@wordpress/components/src/draggable/style.scss';
-@import '~@wordpress/components/src/drop-zone/style.scss';
-@import '~@wordpress/components/src/dropdown/style.scss';
-@import '~@wordpress/components/src/dropdown-menu/style.scss';
-@import '~@wordpress/components/src/external-link/style.scss';
-@import '~@wordpress/components/src/focal-point-picker/style.scss';
-@import '~@wordpress/components/src/font-size-picker/style.scss';
-@import '~@wordpress/components/src/form-toggle/style.scss';
-@import '~@wordpress/components/src/form-token-field/style.scss';
-@import '~@wordpress/components/src/guide/style.scss';
-@import '~@wordpress/components/src/higher-order/navigate-regions/style.scss';
-@import '~@wordpress/components/src/menu-group/style.scss';
-@import '~@wordpress/components/src/menu-item/style.scss';
-@import '~@wordpress/components/src/menu-items-choice/style.scss';
-@import '~@wordpress/components/src/modal/style.scss';
-@import '~@wordpress/components/src/notice/style.scss';
-@import '~@wordpress/components/src/panel/style.scss';
-@import '~@wordpress/components/src/placeholder/style.scss';
-@import '~@wordpress/components/src/popover/style.scss';
-@import '~@wordpress/components/src/radio-control/style.scss';
-@import '~@wordpress/components/src/resizable-box/style.scss';
-@import '~@wordpress/components/src/responsive-wrapper/style.scss';
-@import '~@wordpress/components/src/sandbox/style.scss';
-@import '~@wordpress/components/src/scroll-lock/style.scss';
-@import '~@wordpress/components/src/select-control/style.scss';
-@import '~@wordpress/components/src/snackbar/style.scss';
-@import '~@wordpress/components/src/spinner/style.scss';
-@import '~@wordpress/components/src/tab-panel/style.scss';
-@import '~@wordpress/components/src/text-control/style.scss';
-@import '~@wordpress/components/src/textarea-control/style.scss';
-@import '~@wordpress/components/src/tip/style.scss';
-@import '~@wordpress/components/src/toggle-control/style.scss';
-@import '~@wordpress/components/src/toolbar/style.scss';
-@import '~@wordpress/components/src/toolbar-button/style.scss';
-@import '~@wordpress/components/src/toolbar-group/style.scss';
-@import '~@wordpress/components/src/tooltip/style.scss';
-@import '~@wordpress/components/src/visually-hidden/style.scss';
+@import '~@wordpress/components/build-style/style.css';
```

Comment:
I wonder if now we could just remove these imports and try to use them directly from `@wordpress/components` dependency 🤔. But it would likely be a bigger change for the future. 🙂

Reply:
@renatho  Yes, I tried to do it, but It cause a lot of issues. 😢